bundler/spec/update/gems/fund_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe "bundle update" do
+  let(:config) {}
+
+  before do
+    gemfile <<-G
+      source "#{file_uri_for(gem_repo1)}"
+      gem 'has_metadata'
+      gem 'has_funding', '< 2.0'
+    G
+
+    bundle :install
+  end
+
+  context "when listed gem is updated" do
+    before do
+      gemfile <<-G
+        source "#{file_uri_for(gem_repo1)}"
+        gem 'has_metadata'
+        gem 'has_funding'
+      G
+
+      bundle :update, :all => true
+    end
+
+    it "should display fund message for updated gems" do
```

Comment:
Let's say

```suggestion
    it "displays fund message" do
```

since the current wording makes it unclear whether information is displayed for all gems or only the ones explicitly updated.